src/withStyles.jsx

```diff
@@ -241,20 +247,43 @@ export function withStyles(
       }
     }
 
+    WithStyles.propTypes = {
+      forwardedRef: ref,
+    };
+
+    // TODO: remove conditional once breaking change to only support React 16.3+
+    // ref: https://github.com/airbnb/react-with-styles/pull/240#discussion_r533497857
+    if (typeof React.forwardRef !== 'undefined') {
+      WithStyles.propTypes = {
+        forwardedRef: ref,
+      };
+    }
```

Comment:
Is the first propTypes assignment here a mistake?
```suggestion
    // TODO: remove conditional once breaking change to only support React 16.3+
    // ref: https://github.com/airbnb/react-with-styles/pull/240#discussion_r533497857
    if (typeof React.forwardRef !== 'undefined') {
      WithStyles.propTypes = {
        forwardedRef: ref,
      };
    }
```

Reply:
Good catch! That diff must've snuck in during a rebase.